test/unit/data-types/enum.test.ts

```diff
@@ -0,0 +1,43 @@
+import { DataTypes } from '@sequelize/core';
+import { expect } from 'chai';
+import { sequelize } from '../../support';
+
+describe('DataTypes.ENUM', () => {
+  it('raises an error if no values are defined', () => {
+    expect(() => {
+      sequelize.define('omnomnom', {
+        bla: { type: DataTypes.ENUM },
+      });
+    }).to.throwWithCause(Error, 'DataTypes.ENUM cannot be used without specifying its possible enum values.');
+  });
+
+  describe('setting values for enums', () => {
+    it('should not throw if `values` is a readonly array', () => {
+      const Array = ['A', 'B', 'C', 'D', 'E'] as const;
+      const type = DataTypes.ENUM({ values: Array });
+
+      expect(type.options.values).to.be.equal(Array);
+    });
+
+    it('should not throw if `values` is an mutable array', () => {
```

Comment:
these  tests don't need to expect anything. Mutable/Readonly array is a TypeScript concept, it's not going to cause any runtime error.